templates/services/merge/consensus/lighthouse.tmpl

```diff
@@ -49,9 +49,8 @@
       - --import-all-attestations
       - --metrics
       - --metrics-port={{.ClMetricsPort}}
-      - --metrics-address=0.0.0.0{{if .ClCheckpointSyncUrl}}
-      - --checkpoint-sync-url=${CHECKPOINT_SYNC_URL}{{end}}{{with .CheckpointSyncUrl}}
-      - --checkpoint-sync-url={{.}}{{end}}{{range $flag := .ClExtraFlags}}
+      - --metrics-address=0.0.0.0{{if or .ClCheckpointSyncUrl .CheckpointSyncUrl}}
+      - --checkpoint-sync-url={{if .CheckpointSyncUrl}}{{ .CheckpointSyncUrl }}{{else if .ClCheckpointSyncUrl}}${CHECKPOINT_SYNC_URL}{{end}}{{end}}{{range $flag := .ClExtraFlags}}
```

Comment:
Using `{{else}}` should be enough here. It will work tough.

Reply:
Done